authenticate/views.py

```diff
@@ -0,0 +1,135 @@
+from django.shortcuts import render,redirect
+from django.contrib import messages, auth
+from django.contrib.auth.models import User
+from .forms import RegisterForm, LoginForm, ProfileForm
+from django.http import HttpResponseRedirect
+from django.contrib.auth.decorators import login_required
+
+# Homepage view.
```

Comment:
We need to avoid adding the comments unless they are absolutely necessary or if they are `TODO` comments. The reason is that the comments cannot be verified by unittests and also the developers don't usually update the comments making them lose context in future.

Reply:
Done